libraries/adsk/materials/wood/parquet_3/ND_parquet_03_material.mtlx

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0"?>
 <materialx version="1.37">
-  <nodedef name="ND_parque_03_material" node="parque_03_material" nodegroup="material">
+   <!-- Sample parquet material interface. Versioned and namespaced -->
+   <nodedef name="ND_parque_03_material" node="parque_03_material" nodegroup="material" version="1.0" isdefaultversion="true" namespace="adsk">
```

Comment:
Shouldn't this be called parquet_03_material?

Reply:
Yes. Need to fix my spelling :).